packages/notifications/lib/utils.js

```diff
@@ -0,0 +1,11 @@
+const deleteSessionForDevices = async (tokens, pgdb) => {
+  return pgdb.query(`
+    DELETE FROM sessions s WHERE s.id IN (
+      SELECT d."sessionId" FROM devices d WHERE ARRAY[d.id] && :tokens
```

Comment:
should this really be `d.id`? Tokens are saved in the `device.token` column.

Reply:
👍